AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Itinerary/Details.cshtml

```diff
@@ -125,6 +127,20 @@
         </div>
     </div>
 }
+<form method="post" action="@Url.Action("Details", "Itinerary", new { area="Admin" })#requests" class="form-horizontal">
```

Comment:
@stevejgordon The `#request` is being appended to the URL (and the tag helpers are not being used) so that the posted URL ends up as `/Admin/Itinerary/Details/5#requests`, for example.   I'm adding the target hash because when the filter button is clicked and the page reloads you are left back at the top of the page.  This felt like a bad experience for the user, especially if there are several team members assigned to the itinerary pushing the requests further down the page.  This addition makes sure the the requests are shown at the top of the screen when the page loads. 

I can just add in the older style `@Html.AntiForgeryToken()` inside the form unless you have a suggestion for an alternative solution.